content/agent/_index.md

```diff
@@ -3,103 +3,137 @@ title: Agent
 
-This is all controlled by one supervisor process. We keep this separate so you don't have to have the overhead of each application if you don't want to run all parts, although we generally recommend you do.
+## Agent Status and information
 
-<div class="alert alert-info">
-This documentation covers Agent versions 5.0.0 and above.
-</div>
+### Agent Status
 
-## Configuration management tools
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent status`|`sudo datadog-agent status`|
+|Docker|`sudo docker exec -it dd-agent /etc/init.d/datadog-agent status`||
+|MacOS x|`datadog-agent status`             | `launchctl list com.datadoghq.agent` or systray app|
+|SmartOS|`svcs datadog`|`n/a`|
+|Source|`sudo ~/.datadog-agent/bin/agent status`|`sudo service datadog-agent status`|
+|Windows|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows/#status-and-information)|[Consult our dedicated windows doc](/agent/basic_agent_usage/windows/#status-and-information)|
 
-Manage the Datadog agent and [integrations](/integrations) using configuration management tools:
+### Agent information
 
-### Chef
-* [Chef Github project](https://github.com/DataDog/chef-datadog)
-* [Chef installation](https://app.datadoghq.com/account/settings#integrations/chef)
-* [Chef documentation](/integrations/chef)
+Running an info command displays the status of your Datadog agent, and the status of all integrations enabled for this Agent:
 
-### Puppet
-* [Puppet Github project](https://github.com/DataDog/puppet-datadog-agent)
-* [Puppet installation](https://app.datadoghq.com/account/settings#integrations/puppet)
-* [Puppet documentation](/integrations/puppet)
+An Agent integration reports “OK” if it's properly configured as seen below:
 
-### Ansible
-* [Ansible Github project](https://github.com/DataDog/ansible-datadog)
-* [Ansible installation](https://app.datadoghq.com/account/settings#agent/ansible)
-* [Ansible documentation](/integrations/ansible/)
+```
+  Checks
+  ======
 
-Chef, Puppet, and Ansible [integrations](/integrations) using our public APIs so if you're interested in using another automation tool, the above could be leveraged as examples to get you started.
+    network
+    -------
+      - instance #0 [OK]
+      - Collected 15 metrics, 0 events & 1 service check
+```
 
-There is also community support for Saltstack:
+The `[OK]` in the Agent output implies that the check was configured/run correctly but does not refer to the value being returned by your check.  
 
-* Saltstack Formula - https://github.com/DataDog/datadog-formula
+|Platform|Agent v5 |Agent v6|
+|:--------|:-----|:--------|
+|Linux|`sudo service datadog-agent info`|`sudo datadog-agent status`|
+|Docker|`sudo docker exec -it dd-agent /etc/init.d/datadog-agent info`||
+|Docker (Alpine)|`docker exec -it dd-agent /opt/datadog-agent/bin/agent`||
```

Comment:
an empty cell is ambiguous I think, can we write "Same as Agent v5" or something similar?